score-http/score-http/src/main/java/org/oagi/score/gateway/http/api/context_management/controller/BusinessContextController.java

```diff
@@ -67,16 +77,28 @@ public PageResponse<BusinessContext> getBusinessContextList(
             calendar.add(Calendar.DATE, 1);
             request.setUpdateEndDate(new Timestamp(calendar.getTimeInMillis()).toLocalDateTime());
         }
-
+        request.setTenantId(tenantId);
+        request.setNotConnectedToTenant(notConnectedToTenant != null ? notConnectedToTenant : false);
+        
         request.setPageIndex(pageIndex);
         request.setPageSize(pageSize);
         request.setSortActive(sortActive);
         request.setSortDirection("asc".equalsIgnoreCase(sortDirection) ? ASC : DESC);
 
-        GetBusinessContextListResponse response = businessContextService.getBusinessContextList(request);
+        GetBusinessContextListResponse response = businessContextService.getBusinessContextList(request, configService.isTenantInstance());
 
+        List<BusinessContext> ctxs = response.getResults();
+        if(configService.isTenantInstance()) {
+        	 ctxs.forEach(c->{
```

Comment:
Formatting - the indentation should be decreased

Reply:
> Formatting - the indentation should be decreased

@StefaniPaunova We, NIST team, have used IntelliJ IDE and its default code style scheme. Also, we haven't followed the strict style guide like 'Space instead of Tab char. for indentation', so I think you can use any typical code formatter.